docker/mock-pay/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM golang:1.23.5-alpine AS base
-RUN apk add --no-cache curl=8.11.1-r0 && curl -o /opt/publicapi_spec.json https://raw.githubusercontent.com/alphagov/pay-publicapi/master/openapi/publicapi_spec.json
+RUN apk add --no-cache curl=8.11.1-r0 && curl -o /tmp/publicapi_spec.json https://raw.githubusercontent.com/alphagov/pay-publicapi/master/openapi/publicapi_spec.json
 
 FROM outofcoffee/imposter:4.5.0
 
 COPY ./docker/mock-pay /opt/imposter/config/
 
-COPY --from=base /opt/publicapi_spec.json /opt/imposter/config/
```

Comment:
This container was failing to copy across the spec from `/opt` locally so switched to `/tmp`